src/datachain/lib/dc.py

```diff
@@ -859,6 +859,17 @@ def mutate(self, **kwargs) -> "Self":
                 dist=cosine_distance(embedding_text, embedding_image)
         )
         ```
+
+        This method also has enhanced capabilities for renaming the columns. If column
+        with the current name exists then its renamed. Orelse a new column is created
+        from the existing one.
+
+        Example:
+        ```py
+         dc.mutate(
```

Comment:
do we need a new string here? seems that it's small enough to fit the same line?

Reply:
Can I get more clarity on the same.

Reply:
it's minor, not a blocker:

just to clarify - what I meant is to have 

```python
dc.mutate(newkey=Column("oldkey"))
```

(w/o extra newlines).

Again, not a blocker at all, just a suggestion to consider.